docs/technical_documentation/decisions/0009_pii.rst

```diff
@@ -74,4 +74,4 @@ References
 **********
 
 - `OEP-30: PII Markup and Auditing <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0030-arch-pii-markup-and-auditing.html/>`_
-- `Enabling the User Retirement Feature <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/user_retire/index.html/>`_
+- `Enabling the User Retirement Feature <https://docs.openedx.org/projects/edx-platform/en/latest/references/docs/scripts/user_retirement/docs/index.html/>`_
```

Comment:
Thank you for tracking down this new link @sarina !
Could you remove the trailing slash, though? It breaks the new link.
```suggestion
- `Enabling the User Retirement Feature <https://docs.openedx.org/projects/edx-platform/en/latest/references/docs/scripts/user_retirement/docs/index.html>`_
```